core/src/epicli/data/common/ansible/playbooks/roles/repository/tasks/Debian/setup.yml

```
@@ -4,11 +4,19 @@
     - name: Create epirepo repository
       shell: >-
         set -o pipefail &&
-        /tmp/epi-repository-setup-scripts/create-repository.sh /var/www/html/epirepo {{ offline_mode | lower }} |&
+        /tmp/epi-repository-setup-scripts/create-repository.sh {{ apache_epirepo_path }} {{ offline_mode | lower }} |&
         tee /tmp/epi-repository-setup-scripts/create-repository.log
       args:
         executable: /bin/bash
 
+    - name: Copy helm charts
```

```suggestion
    - name: Copy Helm charts
```
Maybe this task could be run conditionally - skipped when helm role is not enabled?